app/controllers/api/v0/bids_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::V0::BidsController < ApplicationController
+  before_filter :require_authentication, except: [:index]
+  skip_before_action :verify_authenticity_token
+
+  def index
+    auction = AuctionQuery.new.bids_index(params[:auction_id])
+    @auction_bids = BidsIndexViewModel.new(auction: auction, current_user: current_user)
+  end
+
+  def create
+    @bid = PlaceBid.new(params: params, bidder: current_user, via: via)
+
+    if @bid.perform
+      render json: @bid.bid, serializer: BidSerializer
+    else
+      render json: { error: @bid.errors.full_messages.to_sentence }, status: 403
+    end
+  end
+
+  rescue_from 'ActiveRecord::RecordNotFound' do
```

Comment:
I am not sure exactly, but I think we should generally move away from all of the exception rescuing we're doing here. Makes it hard to debug things / find errors